crates/notion-core/src/distro/package.rs

```diff
@@ -123,6 +137,15 @@ pub struct BinConfig {
     pub path: String,
     /// The platform used to install this binary
     pub platform: PlatformSpec,
+    /// The loader information for the script, if any
+    pub loader: Option<BinLoader>,
+}
+
+pub struct BinLoader {
+    /// The loader executable
+    pub exe: String,
```

Comment:
My thought for using `exe` actually wasn't the Windows-only file extension, but rather as a shorthand for `executable`. That said, I agree it isn't an ideal name. My concern with using `bin_name` is that this name is distinct from the `bin_name` in other spots, since it represents the command that is used to run the script, not the script itself (e.g. `node` vs `tsc`).

Maybe `command` would be a better name?

Reply:
I like `command`!

Reply:
Done!